src/routes/api/organisationJobs.route.ts

```diff
@@ -0,0 +1,38 @@
+import express from 'express';
+export const router = express.Router({ mergeParams: true });
+import { check, oneOf } from 'express-validator';
+import { db } from '../../database';
+import { authorise, checkInputError, protect } from '../../middleware';
+import { INVALID_FIELD_MSG, NO_FIELD_UPDATED_MSG } from '../../utils';
+
+// Import controllers and initialise controller
+import { OrganisationJobsController } from '../../controllers/organisationJobs';
+const organisationJobsController = new OrganisationJobsController(db.organisationJobs, db.organisations);
+
+// Validate inputs
+const validateCreateOrganisationJobFields = [
+    check('organisation_id', INVALID_FIELD_MSG('organisation id')).isUUID(),
+    check('job_title', INVALID_FIELD_MSG('job title')).trim().notEmpty(),
+    check('job_description', INVALID_FIELD_MSG('job description')).trim().notEmpty(),
+];
+
+const validateUpdateOrganisationJobFields = [
+    oneOf([check('job_title').exists(), check('job_description').exists()], NO_FIELD_UPDATED_MSG),
+    check('job_title', INVALID_FIELD_MSG('job title')).optional().trim().notEmpty(),
+    check('job_description', INVALID_FIELD_MSG('job description')).optional().trim().notEmpty(),
+];
+
+router.route('/').get(organisationJobsController.getOrganisationJobs);
```

Comment:
From the suggestions in `organisationJobs.ts` controller, `asyncHandler` can be wrapped here as such
```suggestion
router.route('/').get(asyncHandler(organisationJobsController.getOrganisationJobs));
```